packages/devextreme/js/ui/html_editor/ui.html_editor.js

```diff
@@ -189,7 +190,11 @@ const HtmlEditor = Editor.inherit({
         const frameDocument = frameWindow.document;
         const frameDocumentBody = frameDocument.body;
 
-        frameDocumentBody.innerHTML = value;
+        const quill = getQuill();
+
+        frameDocumentBody.innerHTML = quill.replaceStyleAttribute
+            ? quill.replaceStyleAttribute(value)
+            : value;
```

Comment:
I also suggest adding an additional variable just to make this action more noticeable. Maybe even write it directly in the first row of the function:

like
```javascript
  _removeXSSVulnerableHtml: function(value) {
    // NOTE: It fixes "unsafe-inline" csp error.
    const valueWithoutStyles = quill.replaceStyleAttribute(value);
    //...
    frameDocumentBody.innerHTML = valueWithoutStyles;
```

Reply:
thanks. I will fix it